dtcm/standard/coreo/map.xml

```diff
@@ -1,16 +1,24 @@
 <map proto="1.4.2">
 <name>Coreo</name>
-<version>1.1.0</version>
+<version>1.1.1</version>
 <objective>Destroy and leak the enemy's coreos!</objective>
 <authors>
     <author uuid="345e7c1e-2b0f-4b6e-be83-5494ec6d4e7d"/> <!-- ElDanii -->
     <author uuid="dd8c3d79-3930-41f6-a815-621dc3d52649"/> <!-- StanleyTheCow -->
     <author uuid="55cb168a-e28f-49e5-a853-8268093926c8"/> <!-- 0uzi -->
 </authors>
 <contributors>
-    <contributor uuid="5c79d2c9-a4f0-4343-a84b-e1720f13009b" contribution="XML support and suggestions"/> <!-- CoWinkKeyDinkInc, thanks! -->
+    <contributor uuid="5c79d2c9-a4f0-4343-a84b-e1720f13009b" contribution="XML Support and suggestions"/> <!-- CoWinkKeyDinkInc, thanks! -->
 </contributors>
 <gamemode>dtc</gamemode>
+<broadcasts>
+    <tip after="5s">`e`l[Tip] `fGet `loreos `ffrom kill reward.</tip>
+    <tip after="10s">`e`l[Tip] `fBy using `ltwo (2) oreos `fyou can craft a `6Haste III`f potion!</tip>
+    <tip after="50s">`e`l[!] `fRemember to share oreos with your teammates!</tip>
+    <tip after="300s">`e`l[!] `fOreos.. hmm yummy!</tip>
+    <tip after="2m" every="2m">`e`l[Tip] `fGet `loreos `ffrom killrewards.</tip>
+    <tip after="2m" every="2m">`e`l[Tip] `fCraft a `6Haste III`f potion by using `ltwo (2) oreos`f.</tip>
+</broadcasts>
```

Comment:
These are already automatically prefixed with Tip in the chat, you don't need to mention `[Tip]` in the line again.